ui/components/MesheryMeshInterface/PatternServiceForm.js

```diff
@@ -148,6 +150,15 @@ function PatternServiceForm({ formData, schemaSet,onChange, onSubmit, onDelete,
         <AppBar>
           <Toolbar variant="dense" style={{ padding : "0 10px" }}>
             <p style={{ margin : "auto auto auto 10px", fontSize : "18px" }}>{schemaSet.workload.title}</p>
+            {schemaSet?.workload?.description && (
+              <label htmlFor="help-button">
```

Comment:
Does this tag really play any role?

Reply:
Not really sure, I copied this from the codebase

Reply:
Beats me!